src/lib/client.ts

```diff
@@ -184,12 +184,17 @@ export class Client {
   }
 
   public async createRewardAddress(): Promise<string> {
-    const mnemonic = mnemonicGenerate()
-    const keyring = new Keyring({ type: 'sr25519', ss58Format: 2254 }) // 2254 is the prefix for subspace-testnet
-    await cryptoWaitReady();
-    const pair = keyring.createFromUri(mnemonic)
-    this.mnemonic = mnemonic
-    return pair.address
+    try {
+      const mnemonic = mnemonicGenerate()
+      const keyring = new Keyring({ type: 'sr25519', ss58Format: 2254 }) // 2254 is the prefix for subspace-testnet
+      await cryptoWaitReady();
+      const pair = keyring.createFromUri(mnemonic)
+      this.mnemonic = mnemonic
+      return pair.address
+    } catch (error) {
+      util.errorLogger(error)
+      return "???"
```

Comment:
do we actually display this in case of error?

Reply:
Let me update this with `""` instead, that will make more sense, because there are checks for `rewardAddress !== ""`. Will be better for logging. Thanks